changes_revision.go

```diff
@@ -140,6 +141,35 @@ type PatchOptions struct {
 	Path string `url:"path,omitempty"`
 }
 
+// StringPointerWriter is a new type based on *string.
+// We would like it to implement Writer interface so that we can avoid
+// unmarshalling of non JSON responses by Client.Do
+type StringPointerWriter struct {
```

Comment:
This is an implementation detail of this package, it shouldn't be made exported. Once it's exported, it can't be removed without breaking API compatibility.